src/processTargets/getModifierStage.ts

```diff
@@ -63,8 +39,9 @@ export default (modifier: Modifier): ModifierStage => {
     case "trailing":
       return new TrailingStage(modifier);
     case "containingScope":
```

Comment:
Noticed that containing and every are two separate modifiers. The different scope types are not modifiers per se.